product_docs/docs/biganimal/release/using_cluster/06_demonstration_oracle_compatibility.mdx

```diff
@@ -238,8 +238,70 @@ In both of the examples shown here, you probably would not use the functions and
 better, more familiar or at least more widely-available equivalents provided natively by PostgreSQL (and many other databases). But by supporting them, EDB Advanced Server gives you the ability to reuse existing logic with minimal modification, allowing
 you to focus your time and expertise on solving new problems.
 
+## Try it on your own cluster: export and import
+
+If you'd like to try these examples on your own BigAnimal cluster, follow these instructions to import the example database.
+
+1. [Connect to your EDB Postgres Advanced Server cluster as edb_admin](https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/02_connecting_your_cluster/) using psql.
+
+2. Create a new database and role:
+
+    ```sql
+    CREATE DATABASE chinook;
+    \c chinook
+    CREATE USER demo WITH PASSWORD 'password';
+    ALTER DEFAULT PRIVILEGES IN SCHEMA public GRANT SELECT ON TABLES TO demo;
+    ```
+
+    See [Details on managing access in BigAnimal databases](https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/01_postgres_access/) for more information.
+
+3. Quit psql (`\q`).
+
+4. To export and import, you'll need compatible versions of pg_dump and pg_restore. If you aren't already running EDB Postgres Advanced Server, you can use the container image used by Big Animal to ensure compatibility.
+
+    ```shell
+    docker pull quay.io/enterprisedb/edb-postgres-advanced
+    ```
+
+5. Export the Chinook sample database using run pg_dump:
+
+    ```shell
+    docker run --rm quay.io/enterprisedb/edb-postgres-advanced:latest \
+        pg_dump --format custom \
+        "postgres://demo:password@p-xj6qj8i1hx.qsbilba3hlgp1vqr.biganimal.io:5432/chinook?sslmode=require" \
```

Comment:
It's a bit confusing that we're connecting to another cluster using the same username/password. I initially wondered if I was supposed to change the connection string to get it to work.

Reply:
maybe I can be more explicit about this; "export from EDB's cluster"?